01week/rockPaperScissors.js

```diff
@@ -12,6 +12,22 @@ function rockPaperScissors(hand1, hand2) {
 
   // Write code here
 
+  
+  hand1 = hand1.toLowerCase().trim();
+  hand2 = hand2.toLowerCase().trim();
+
```

Comment:
Consider breaking your validation check into another function.  It be beneficial later when you are writing more complicated applications.